e2e/nomostest/testpredicates/predicates.go

```diff
@@ -1171,3 +1171,51 @@ func RootSyncSpecOverrideEquals(expected *v1beta1.RootSyncOverrideSpec) Predicat
 		return nil
 	}
 }
+
+func subjectsEqual(want []string, got []rbacv1.Subject) error {
```

Comment:
Not exactly, since this function is only comparing the name field of each `Subject`. We would still need to convert the data structures to look the same in order to use `cmp.Equal`